.gitignore

```diff
@@ -17,8 +17,6 @@ extensions
 /facades/*
 !/facades/PC
 !/facades/subprojects.gradle
-/modules/*
-!/modules/subprojects.gradle
```

Comment:
We have a multi-repo workspace setup, so this change is not needed - on the contrary.
Please have a look at https://terasology.org/Terasology/#/Developing-Modules?id=understanding-terasology39s-git-setup and https://terasology.org/Terasology/#/Multi-Repo-Workspace to understand our workspace setup.